help/contributor/how-to/frontmatter.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Front matter"
+linkTitle: ""
+weight: 30
+type: "docs"
+---
+
+The front matter is YAML code in between triple-dashed lines at the top of each
+file and provides important management options for our content. For example, the
+front matter allows us to ensure that existing links continue to work for pages
+that are moved or deleted entirely. This page explains the front matter
+features that are currently available in knative.dev.
+
+The following example shows a front matter with all the required fields
+filled by placeholders:
+
+```
+---
+title: <title>
+description: <description>
+weight: <weight>
+keywords: [<keyword1>,<keyword2>,...]
+aliases:
+    - <previously-published-at-this-URL>
+---
+```
```

Comment:
Yes. By default, Hugo uses the Root folder name to determine what "template" to use to build the content (for example Blog has some special blog formatting, so those files should be type: "blog"). 

To simply a release, we include type: "docs" on all pages because we move all the files in a release into a separate "v0.##-docs" folder (created from each versions separate branch). Since those folders cant be named "docs", we need to tell Hugo what template to use.